INSTALL.md

```diff
@@ -11,6 +11,26 @@ PyBitmessage can be run in either one of two ways:
 - from an installed
 package.
 
+## setuptools
+This is now the recommended and in most cases the easiest procedure for installing PyBitmessage. There is also a helper script to resolve dependencies easily.
+
+Go to the directory with PyBitmessage source code and run:
+```
+python checkdeps.py
+```
+If there are missing dependencies, it will explain you what is missing and for many Unix-like systems also what you have to do to resolve it. How you then run setuptools depends on whether you want to install it to user's directory or system.
+
+As normal user:
+```
+python setup.py install --user
+~/.local/bin/pybitmessage
+```
+or as root:
+```
+python setup.py install
+/usr/local/bin/pybitmessage
+```
+
```

Comment:
missing reference to venv (other than activating the venv, process is the same as when using setuptools as root)

Reply:
added virtualenv instructions

Reply:
updated